src/style.css

```diff
@@ -1 +1,80 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap");
+
+html {
+  margin: inherit;
+  padding: 0;
+}
+
+.header {
+  min-width: 1024px;
+  max-width: 1200px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  height: 60px;
+  align-items: center;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  padding-right: 50px;
+}
+
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.MOYOimg {
+  margin-left: 20px;
+  float: left;
+  margin-bottom: 10px;
+}
+
+.nav_list {
+  list-style: none;
+  margin: 0;
+  font-size: 0.7em;
+  display: flex;
+  text-transform: uppercase;
+}
+
+.nav_item {
+  font-weight: 700;
+  padding: 5px 20px;
+  display: flex;
+}
+
+.is-active:focus::after {
+  position: relative;
+  top: 10.5px;
+  display: block;
+  content: "";
+  width: 100%;
+  height: 4px;
+  background-color: #00acdc;
+  border-radius: 8px;
+}
+
+.nav_links {
+  text-decoration: none;
+  color: black;
+  padding: 10px 0;
+}
+
+.nav_links:hover {
+  color: #00acdc;
+}
+
+.nav_links:focus {
+  color: #00acdc;
+}
```

Comment:
You shouldn't add any styles on focus state
Your header should has only one static active link (the first one) with specified styles